openmetadata-service/src/main/resources/elasticsearch/zh/classification_index_mapping.json

```diff
@@ -98,6 +98,12 @@
       "disabled" : {
         "type": "text"
       },
+      "totalVotes": {
+        "type": "long"
```

Comment:
@mohityadav766 we should add "null_value": "0"

Reply:
addressed